foia_hub/scripts/load_agency_contacts.py

```diff
@@ -31,80 +31,66 @@ def check_urls(agency_url, row, field):
         return row_url
 
 
-TTY_RE = re.compile('\(?\d{3}\)? \d{3}-\d{4} \(TTY\)')
+TTY_RE = re.compile('\d{3}-\d{3}-\d{4} \(TTY\)')
```

Comment:
All numbers should be formatted properly at this point. Is there a reason for keeping the regex? Also, what if a TTY number has an extension?